src/taipy/core/_repository/_sql_repository.py

```diff
@@ -51,7 +51,10 @@ def _save(self, entity: Entity):
             return
         self.__insert_model(obj)
 
-    def _load(self, entity_id: Any) -> Entity:
+    def _exists(self, entity_id: str):
+        return bool(self.db.query(self.model_type).filter_by(id=entity_id).first())
```

Comment:
Instead of using a select method, did you try to use the exists method? It should be faster since it s designed for that.
https://docs.sqlalchemy.org/en/14/orm/query.html#sqlalchemy.orm.Query.exists

Reply:
In https://stackoverflow.com/questions/32938475/flask-sqlalchemy-check-if-row-exists-in-table, there is also an example that are using `exists()` method.

However, that requires a subquery, which (slightly) worsen the performance.
It goes something like this
```suggestion
        return self.db.query(
            self.db.query(self.model_type).filter_by(id=entity_id).exists()
        ).scalar()
```

Reply:
If we want to improve the performance a bit, instead of querying the entire model, we can only query the id.
```suggestion
        return bool(self.db.query(self.model_type.id).filter_by(id=entity_id).first()) # type: ignore
```

Reply:
And what about just that ?
```
return self.db.query(self.model_type.id).filter_by(id=entity_id).exists()
```

Reply:
That will returns a query. To execute that query, we need to wrap it inside another `self.db.query(...)`.

Reply:
I'm not an expert in SQL, so maybe we can have a third opinion about this.